packages/dara-core/js/shared/interactivity/persistence.tsx

```diff
@@ -0,0 +1,307 @@
+import { mixed } from '@recoiljs/refine';
+import * as React from 'react';
```

Comment:
In most places we have been importing what was needed instead of doing React.useSomething. Is this better?

Reply:
Recently we've been doing both interchangeably, doesn't matter too much - personal preference. Both are fine and should be handled by bundlers to treeshake correctly.